src/groovy/au/org/emii/aatams/JodaOverrides.groovy

```diff
@@ -0,0 +1,42 @@
+package au.org.emii.aatams
+
+import org.joda.time.Interval
+import org.joda.time.base.AbstractInterval
+
+class JodaOverrides {
+
+    // Would be nicer to do this with an "Extension Module", but alas, that's
+    // only available in groovy 2.0.
+    //
+    // An extension module would avoid the need to mock/unmock in every test
+    // which executes the overlapsAny code.
+    static void apply() {
+        AbstractInterval.metaClass.contains = { List instantList ->
+            def owningInterval = delegate
+
+            null != instantList.find {
+                it && owningInterval.contains(it)
+            }
```

Comment:
I can change it.

It could probably simply be:

```groovy
instantList.find {
    it && owningInterval.contains(it)
}
``` though (supposing java and groovy play well together there).